### PR TITLE
[Admin] [Customer]Remove delete button on index, add information about no account on show page

### PR DESCRIPTION
**Pull request by tuka217**
| Q               | A
| --------------- | ---
| Bug fix?        | yes
| New feature?    | yes
| BC breaks?      | no
| Related tickets | -
| License         | MIT

### CODE REVIEW

File: features/account/sign_in.feature [hunk 1]
```diff
@@ -24,3 +24,13 @@ Feature: Sign in to the store
         And I log in
         Then I should be notified about bad credentials
         And I should not be logged in
+
+    @todo
+    Scenario: A customer cannot log in after deleting their account
```
Comment (TheMadeleine) [inline features/account/sign_in.feature]: after his account was deleted?

File: features/account/sign_in.feature [hunk 2]
```diff
@@ -24,3 +24,13 @@ Feature: Sign in to the store
...
+        Given the account of "ted@example.com" was deleted
+        When this customer try to log in
+        And he specify the username as "theodore@example.com"
+        And he specify the password ass "pswd"
+        And he log in
```
Comment (TheMadeleine) [inline features/account/sign_in.feature]: tries, specifies, logs in ;D third forms please.

Comment (Niiko) [inline features/account/sign_in.feature]: the password ass :D

File: features/account/sign_in.feature (hunk 1 above)
Comment (tuka217) [inline features/account/sign_in.feature]: 1. Cannot log in after account deletion/removal
2. Cannot log in after my account was deleted  ?

Comment (TheMadeleine) [inline features/account/sign_in.feature]: Second one :+1:. The first one may suggest that it was deleted by the customer himself. :)

File: features/user/managing_customers/seeing_customer_details.feature [hunk 3]
```diff
@@ -22,3 +22,8 @@ Feature: Seeing customer's details
         When I view details of the customer "f.baggins@shire.me"
         Then his shipping address should be "Frodo Baggins, Bag End, Hobbiton, NEW ZEALAND 1"
         And his billing address should be "Bilbo Baggins, The Last Homely House, Rivendell, NEW ZEALAND 7"
+
+    @ui
+    Scenario: Seeing information about no existing account for given customer
```
Comment (TheMadeleine) [inline features/user/managing_customers/seeing_customer_details.feature]: a given customer

File: features/user/managing_customers/seeing_customer_details.feature [hunk 4]
```diff
@@ -22,3 +22,8 @@ Feature: Seeing customer's details
...
+        When I view details of the customer "f.baggins@shire.me"
+        Then I should see information about no existing account
```
Comment (TheMadeleine) [inline features/user/managing_customers/seeing_customer_details.feature]: no existing account for this email?

Comment (tuka217) [inline features/user/managing_customers/seeing_customer_details.feature]: It was `for this customer`,  but for me it is obvious that there is no account for this customer/user, as soon as we specify customer in previous step, WDYT?

**Comment by pjedrzejewski** [comment]
Thanks Ania!

**Pull request merged**